src/Builder/JsonExpressionBuilder.php

```diff
@@ -39,13 +38,11 @@ public function __construct(private QueryBuilderInterface $queryBuilder)
      * @throws NotSupportedException
      *
      * @return string The raw SQL that won't be additionally escaped or quoted.
-     *
-     * @psalm-param JsonExpression $expression
      */
-    public function build(ExpressionInterface $expression, array &$params = []): string
+    public function build(JsonExpression $expression, array &$params = []): string
     {
         /**
-         * @psalm-var array|mixed|QueryInterface $value
+         * @psalm-var array|mixed|QueryInterface|ArrayExpression $value
          */
         $value = $expression->getValue();
```

Comment:
`$value` is mixed according to this code: https://github.com/yiisoft/db/blob/master/src/Expression/JsonExpression.php#L32